financial-connections/src/main/java/com/stripe/android/financialconnections/ui/components/TopAppBar.kt

```diff
@@ -201,27 +199,6 @@ private fun Title(hideStripeLogo: Boolean, testmode: Boolean) = Row(
     }
 }
 
-/**
- * calculates toolbar elevation based on [ScrollState]
- */
-internal val ScrollState.elevation: Dp
-    get() {
-        return minOf(value.dp, AppBarDefaults.TopAppBarElevation)
-    }
-
-/**
- * calculates toolbar elevation based on [LazyListState]
- */
-internal val LazyListState.elevation: Dp
-    get() = if (firstVisibleItemIndex == 0) {
-        // For the first element, use the minimum of scroll offset and default elevation
-        // i.e. a value between 0 and 4.dp
-        minOf(firstVisibleItemScrollOffset.toFloat().dp, AppBarDefaults.TopAppBarElevation)
```

Comment:
Removing this specific elevation logic, as we now just use `animateDpAsState` to animate between elevated and not-elevated.